index.html

```diff
@@ -1086,6 +1092,120 @@ <h3 id="applying">
       </p>
       <section>
         <h3>
+          <code>dir</code> member
+        </h3>
+        <p>
+          The <dfn><code>dir</code> member</dfn> specifies the <dfn>default
+          base direction</dfn> for the <a>directionality-capable members</a> of
+          the <a>manifest</a>. The <code>dir</code> member's value can be set
+          to one of the <a>text-direction values</a>.
+        </p>
+        <p>
+          The <dfn>directionality-capable members</dfn> are:
+        </p>
+        <ul>
+          <li>
+            <a><code>name</code> member</a>.
+          </li>
+          <li>
+            <a><code>short_name</code></a> member.
+          </li>
+        </ul>
+        <p>
+          The <dfn>text-direction values</dfn> are the following, implying that
+          the value of the <a>directionality-capable members</a> is by default:
+        </p>
+        <dl>
+          <dt>
+            <dfn data-lt="direction ltr">ltr</dfn>
+          </dt>
+          <dd>
+            left-to-right text.
+          </dd>
+          <dt>
+            <dfn data-lt="direction rtl">rtl</dfn>
+          </dt>
+          <dd>
+            right-to-left text.
+          </dd>
+          <dt>
+            <dfn data-lt="direction auto">auto</dfn>
+          </dt>
+          <dd>
+            <a data-lt=
+            "steps to programmatically determine the directionality of a member">
+            programmatically determined</a> from the value of the member.
+          </dd>
+        </dl>
+        <p>
+          When displaying the <a>directionality-capable members</a> to an
+          end-user, the user agent MUST use the <a>default base direction</a>
+          to correctly position the characters in text containing characters
```

Comment:
"correctly position the characters in text containing characters flowing from right to left" is not quite correct. It isn't necessarily about positioning, although one might refer to it that way.

Perhaps:
--
... to compute directional runs and layout or position text correctly in text containing mixed-direction sequences.
--